daemon/src/model/cfd.rs

```diff
@@ -2577,6 +2707,30 @@ mod tests {
             ]
         }
 
+        fn dummy_ongoing_collab_settlement(order_id: OrderId) -> Vec<Self> {
```

Comment:
nit: I had a hard time reading what would "ongoing" mean - did you try say that it got started? maybe it's easier to say:
`apply_dummy_collab_settlement_started_event` or something similar?

Reply:
yeah, the `Event` extension should not know about how this is used in the test so started makes more sense, because that's the only terminology the `Event` knows. 

From the test's perspective started might not be enough to understand why the test was written. I tried to convey "started but not finished" with ongoing... Not sure it actually helps.